ph5/utilities/tests/test_metadatatoph5.py

```diff
@@ -1,47 +1,48 @@
 '''
 Tests for metadatatoph5
 '''
-
+import os
+import sys
 import unittest
-from ph5.utilities import metadatatoph5
-from ph5.utilities import initialize_ph5
+
 from obspy.core import inventory
 from obspy import UTCDateTime
-import os
-import sys
 from mock import patch
-from ph5.core.tests.test_base import initialize_ex
+from testfixtures import OutputCapture
 
+from ph5.utilities import metadatatoph5
+from ph5.core.tests.test_base import LogTestCase, TempDirTestCase,\
+    initialize_ex
 
-class TestMetadatatoPH5(unittest.TestCase):
 
+class TestMetadatatoPH5(TempDirTestCase, LogTestCase):
     def setUp(self):
-        self.path = 'ph5/test_data/miniseedph5'
-        try:
-            os.mkdir(self.path)
-        except OSError:
-            print ("Creation of the directory %s failed" % self.path)
-        else:
-            print ("Successfully created the directory %s " % self.path)
+        super(TestMetadatatoPH5, self).setUp()
+        if self._testMethodName != 'test_main':
```

Comment:
Put `test_main` in another class without this setUp it doesn't need.

Reply:
will have another class for test_main.